resources/js/components/Create.vue

```diff
@@ -88,8 +98,19 @@
                 this.idForTodoTitle='';
                 this.newTodoBody = '';
                 this.idForTodoBody='';
-            }
+            },
 
+            remove(id){
+                let confirmed = confirm('Are you sure you want to delete?');
+                if (confirm){
```

Comment:
You're using an undefined variable in the if statement. The variable above is named `confirmed`, not `confirm`.

Also as the value of `confirmed` won't be changing after it's been assigned, it's best to use `const` instead of `let`.

Also, just for code formatting standards, there should be a blank space after the final parenthesis in the if statement

e.g. `if (confirmed) {`